lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -190,7 +190,7 @@ export abstract class CacheManager implements ICacheManager {
      * saves a cache record
      * @param cacheRecord
      */
-    saveCacheRecord(cacheRecord: CacheRecord): void {
+    async saveCacheRecord(cacheRecord: CacheRecord): Promise<void> {
```

Comment:
Is this a breaking change?

Reply:
Correct me if I'm wrong, but `saveCacheRecord` is only used internally by the `ResponseHandler` in common, so this isn't a breaking change. However, do we need to update the method signature in `ICacheManager`? If so, I think that would constitute a breaking change. Although it doesn't seem that our platform libs implement `saveCacheRecord` since common has all the logic.